3.2/Dockerfile

```diff
@@ -67,12 +67,15 @@ RUN buildDeps=' \
 		libsqlite3-dev \
 		make \
 		patch \
+		libgmp-dev \
+		build-essential \
```

Comment:
Using `build-essential` is always a red flag for me -- here's a quote from the package description (https://packages.debian.org/stretch/build-essential):

> If you do not plan to build Debian packages, you don't need this package.

There are usually more specific alternatives that are more appropriate. :+1:

Reply:
I've just tested locally, and I don't think any of these package additions are necessary -- the installation works fine for me with just adding `sqlserver` to the list of adapters below! :+1:

Reply:
Understood, I'll yank it, see what fails, and replace it with more specific packages.